### PR TITLE
Basic reimplementation

### PR DESCRIPTION
**Pull request by richfitz**
Uses orderly3, with the code here: https://github.com/mrc-ide/orderly3/pull/8

### CODE REVIEW

File: tests/testthat/examples/interpolate/orderly.R [hunk 1]
```diff
@@ -0,0 +1,7 @@
+orderly3::orderly_parameters(mpg_min = NULL)
+orderly3.db::orderly_db_query(
+  query = "SELECT * FROM mtcars WHERE mpg > ?mpg_min",
```
Comment (r-ash) [inline tests/testthat/examples/interpolate/orderly.R:3]: Nice! Worth documenting this `?name` syntax somewhere? Maybe in `orderly_db_query` and `orderly_db_view`?

**Review by r-ash — APPROVED** [review]
This looks great too, small comment about some possibly missing docs

File: tests/testthat/examples/interpolate/orderly.R (hunk 1 above)
Comment (richfitz) [reply tests/testthat/examples/interpolate/orderly.R:3]: yeah - this is from the original orderly, I'll get some docs transferred over

**Review by r-ash — CHANGES_REQUESTED** [review]
This looks good, do you want to add a bit about how to use parameters in the query?

**Comment by richfitz** [comment]
So, it looks like we never actually documented that in orderly v1, I've added a note in d46cd58 here

**Review by r-ash — APPROVED** [review]
Looks good!

**Review by weshinsley — APPROVED** [review]
Read and tested - think my understanding will solidify more when I start writing more orderly3 reports... 

(I had never noticed the ?var interpolation syntax before...)

**Pull request merged**